app.js

```diff
@@ -1,21 +1,40 @@
 const API_URL = 'https://cors-anywhere.herokuapp.com/http://moviefinder.us-east-1.elasticbeanstalk.com/';
 
+function redirectPost(url, id) {
```

Comment:
Really happy to see that this function is ready.  I will be using it in the functions that I am working on this afternoon.